src/webdriver.nim

```diff
@@ -250,22 +258,20 @@ proc internalExecute(self: Session, code: string, args: varargs[JsonNode], kind:
   let reqUrl = $(self.driver.url / "session" / self.id / "execute" / kind)
   let obj = %*{
     "script": code,
-    "args": []
+    "args": args
   }
-  for arg in args:
-    obj["args"].elems.add arg
 
   let resp = self.driver.client.post(reqUrl, $obj)
   let respObj = checkResponse(resp.body)
-  if respObj["value"].hasKey("error"):
+  if respObj["value"].kind == JObject and respObj["value"].hasKey("error"):
     raise newException(JavascriptException, respObj["value"]["message"].getStr & "\n" & respObj["value"]["stacktrace"].getStr)
 
   return respObj["value"]
 
-proc execute*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
+proc execute*(self: Session, code: string, args: varargs[JsonNode, `%`]): JsonNode =
   self.internalExecute(code, args, "sync")
 
-proc executeAsync*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
+proc executeAsync*(self: Session, code: string, args: varargs[JsonNode, `%`]): JsonNode =
```

Comment:
Implicitly converting any argument to a JSON node seems risky here. I'd prefer if the user is explicit.

Reply:
Sure. I'll make two explicit procs for the `Element` object